Source/WebCore/platform/LengthPoint.h

```diff
@@ -32,36 +32,17 @@ namespace WebCore {
 struct BlendingContext;
 
 struct LengthPoint {
-public:
-    LengthPoint()
-    {
-    }
-    
-    LengthPoint(Length x, Length y)
-        : m_x(WTFMove(x))
-        , m_y(WTFMove(y))
-    {
-    }
```

Comment:
The `Length` structure consists of a union (packed into an integer), an 8-bit type, and two bools. The copy operation is probably super fast, so these move operations were never useful.